From 1835501c818e549011747055acf10ada408aa9ea Mon Sep 17 00:00:00 2001
From: CI Assistant <yossi.ilkanaev@intel.com>
Date: Mon, 7 Jan 2019 04:17:36 +0100
Subject: [PATCH] Merge pull request #374 in SW_UGW/linux from
 bugfix/DRVLIB_SW-1090-replace-falconmx-ltq_system_reset to master

* commit '3cb9b957420a8ea9c7d4b87592ad9bbcad0a0bbd':
  DRVLIB_SW-1090 - ltq_system_reset: replace falconmx with prx300
---
 drivers/char/ltq_reset/ltq_system_reset.c | 2 +-
 include/linux/ltq_system_reset.h          | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/char/ltq_reset/ltq_system_reset.c b/drivers/char/ltq_reset/ltq_system_reset.c
index 6d5b51da1ebe..7c3db9913059 100644
--- a/drivers/char/ltq_reset/ltq_system_reset.c
+++ b/drivers/char/ltq_reset/ltq_system_reset.c
@@ -2521,7 +2521,7 @@ static int sysrst_remove(struct platform_device *pdev)
 
 static const struct of_device_id sysrst_match[] = {
 	{ .compatible = "intel,sysrst-xrx500" },
-	{ .compatible = "intel,sysrst-falconmx" },
+	{ .compatible = "intel,sysrst-prx300" },
 	{},
 };
 MODULE_DEVICE_TABLE(of, sysrst_match);
diff --git a/include/linux/ltq_system_reset.h b/include/linux/ltq_system_reset.h
index 8484fddf8370..ae8c7c1d6e26 100644
--- a/include/linux/ltq_system_reset.h
+++ b/include/linux/ltq_system_reset.h
@@ -230,7 +230,7 @@ enum {
 	SYSRST_DOMAIN_PCIE2,
 	SYSRST_DOMAIN_CPU_CLUSTER_RST,
 	SYSRST_DOMAIN_CPU_CLUSTER_RLS,
-	/* extension for falconmx */
+	/* extension for prx300 */
 	SYSRST_DOMAIN_GPHY,
 	SYSRST_DOMAIN_PONIP,
 	SYSRST_DOMAIN_HYST0,
